### DISCUSSION TITLE
How to run ps:scale again after using app.json in the past

### DISCUSSION THREAD

**Discussion by lukaszb** [dokku/dokku#4957] [Q&A] [answered]
I've used `app.json` in the past to control number of instances my service should spin. This however ended up to be problematic to keep after resources on the host machine started to get fully used. I run production and staging environments at the same host and wanted to scale those apps differently. So I removed `app.json`, pushed/depoyed changes and just wanted to use `dokku ps:scale` - this is what I'm seeing now:

    App myapp-dev contains an app.json file with a formations key and cannot be manually scaled

`app.json` is not at the repo anymore but I understand dokku caches this strongly through some properties:

    vbee# ~ dokku ps:report myapp-dev
    =====> myapp-dev ps information
          Deployed:                      true
          Processes:                     2
          Ps can scale:                  false
          Ps computed procfile path:     Procfile
          Ps global procfile path:       Procfile
          Ps procfile path:
          Ps restart policy:             on-failure:10
          Restore:                       true
          Running:                       true
          Status web 1:                  running (CID: 8fea930370d)
          Status worker 1:               running (CID: 961416a8cea)

Is there any way I can tell dokku to rollback those properties and let me manually scale the app again with `dokku ps:scale myapp-dev web=X` ?

**Comment by josegonzalez** [comment]
Can you push the app again without the app.json file? It should fetch the new config and therefore re-check the values. If not, that is definitely a bug.

**Reply by lukaszb** [reply]
Yes, I tried that. Forgot to mention.

**Comment by john-doherty** [comment, marked as answer]
I think this is just a misleading error message, I just ran `dokku ps:scale myapp web=4` on dokku version `0.26.6` and got the error:

`App myapp-dev contains an app.json file with a formations key and cannot be manually scaled`

I have never used an `app.json` file, but I did have a `DOKKU_SCALE` file in the root of my project, so I removed that, pushed the change and ran`dokku ps:scale myapp web=4` again and it worked.

**Reply by lukaszb** [reply]
@john-doherty omg, yes, that was it, thanks! After removing `DOKKU_SCALE` I can scale the app with `dokku ps:scale`.
